src/command/install/plugin.rs

```diff
@@ -5,11 +5,21 @@ use apollo_federation_types::config::{FederationVersion, PluginVersion, RouterVe
 use binstall::Installer;
 use camino::Utf8PathBuf;
 use rover_std::{sanitize_url, Fs};
-use semver::Version;
+use semver::{BuildMetadata, Prerelease, Version};
 use serde::{Deserialize, Serialize};
 
 use crate::{utils::client::StudioClientConfig, RoverError, RoverErrorSuggestion, RoverResult};
 
+// The first version of the router
+// That was compiled for aarch64 only
+const AARCH_OSX_FIRST_ROUTER_VERSION: Version = Version {
+    major: 1,
+    minor: 38,
+    patch: 0,
+    pre: Prerelease::EMPTY,
+    build: BuildMetadata::EMPTY,
+};
```

Comment:
```suggestion
const AARCH_OSX_FIRST_ROUTER_VERSION: Version = Version::new(1, 38, 0);
```
`Version::new` is a const fn in latest `semver`, unless we're not using that yet?

Reply:
great catch ,didnt notice if was a const fn \o/

Reply:
fixed in a41eefec